ts/packages/shell/src/renderer/src/chatInput.ts

```diff
@@ -93,8 +93,9 @@ export function questionInput(
         "replacementDiv",
         "replacement-textarea",
         {
-            onSend: (text) => {
-                chatView.answer(questionId, text, id);
+            onSend: (html) => {
+                // REVIEW: text is from innerHTML, is that ok?
```

Comment:
@robgruen this change fixed a regression from the vision changes (where the display start showing HTML stuff as plain text like &nbsp;) because it didn't expect HTML content that doesn't start with "<".   

However, the `answer` will get HTML content instead.  I added a "REVIEW" here.  just FYI.